core/tauri-runtime-wry/src/lib.rs

```diff
@@ -2734,10 +2735,21 @@ fn create_webview<T: UserEvent, F: Fn(RawWindow) + Send + 'static>(
     .unwrap() // safe to unwrap because we validate the URL beforehand
     .with_transparent(is_window_transparent)
     .with_accept_first_mouse(webview_attributes.accept_first_mouse);
+
   if webview_attributes.file_drop_handler_enabled {
-    webview_builder = webview_builder
-      .with_file_drop_handler(create_file_drop_handler(window_event_listeners.clone()));
+    let proxy = proxy.clone();
+    let window_event_listeners = window_event_listeners.clone();
+    webview_builder = webview_builder.with_file_drop_handler(move |event| {
+      let event: FileDropEvent = FileDropEventWrapper(event).into();
+      let _ = proxy.send_event(Message::Webview(
+        window_id,
+        WebviewMessage::WebviewEvent(WebviewEvent::FileDrop(event)),
+      ));
+      let has_listeners = !window_event_listeners.lock().unwrap().is_empty();
```

Comment:
I think we should always return true now, we can't know for sure if the event was handled or not (that was the case with the previous implementation too - which is also wrong on v1)

Reply:
unless... we add a new API (I'll take a look)

Reply:
An API would be nicer indeed

Reply:
as I suspected we can't really determine this unless we create a closure just to handle the file drop event, and remove it from the event loop event enum. yikes. I think we should always block the default OS behavior instead.